src/components/Tutorial.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Card from './Card';
+
+import './Tutorial.css';
+
+const Tutorial = ({ tutorial, issueNumber }) => {
+  return (
+    <li className="tutorial" aria-label="Tutorial list item">
+      <Card
+        header={
+          <>
```

Comment:
I think the complexity of the embedded html starts to make readability of the `Card` component stretch a bit thin.

It's fine and I don't have a better solution off the top of my head but it's something to consider if you find yourself adding more complexity to it.